test/path_filtering_test.go

```diff
@@ -103,3 +103,50 @@ func TestAlphaPathFilter(t *testing.T) {
 		})
 	}
 }
+
+func TestBetaPathFilter(t *testing.T) {
+	for _, tc := range []struct {
+		path    string
+		allowed bool
+	}{{
+		path:    "/test.yaml",
+		allowed: true,
+	}, {
+		path:    "/alpha/test.yaml",
```

Comment:
Do paths ever contain "alpha" (i.e. missing a slash at the beginning or end)?

Reply:
I don't think so. These cases are the same as those from the stablePath tests. 
If we do encounter a case, we can update the filters later. But I doubt it.